core/adapter/btc-usdt.adapter.json

```diff
@@ -1,241 +1,303 @@
 {
   "id": "0x193a5f196fce5b0cc022f7c298b71d425886e062a3f3401db68f7df4ae6ed9c1",
-  "active": true,
-  "name": "BTC/USDT",
-  "jobType": "DATA_FEED",
+  "name": "BTC-USDT",
   "decimals": 8,
   "feeds": [
     {
-      "url": "https://api.bybit.com/derivatives/v3/public/tickers?symbol=BTCUSDT",
-      "headers": {
-        "Content-Type": "application/json"
-      },
-      "method": "GET",
-      "reducers": [
-        {
-          "function": "PARSE",
-          "args": ["result", "list"]
+      "name": "Bybit-ticker-BTC-USDT",
```

Comment:
General structure in naming of feeds looks like `CEXName-X-Y`
Is there any reason why we put `ticker` in only `Bybit` ?

Reply:
This is good finding! It is mistake.

At some point I thought there were more queries than one for Bybit, therefore I wanted to distinguish between them. Then, I realized I was wrong but did not fix this one. If you have time, can you please make a bugfix for this?

Reply:
Sure. I will create an issue and fix it.